<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1167</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabs">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>1131</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="usesScrollButtons">
    <bool>true</bool>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Render nodes</string>
    </attribute>
    <widget class="QTableWidget" name="scanline_table">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>661</width>
       <height>301</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>80</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>30</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Scanline Render</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Filter</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Antialiasing</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Samples</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Shutter</string>
      </property>
     </column>
    </widget>
    <widget class="QTableWidget" name="ray_table">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>0</y>
       <width>451</width>
       <height>301</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>80</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>30</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Ray Render</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Filter</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Samples</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Shutter</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Write nodes</string>
    </attribute>
    <widget class="QTableWidget" name="write_table">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1121</width>
       <height>301</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>80</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>30</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Write node</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Output directory</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Comment</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Padding</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Format</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Range</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QTextEdit" name="log">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>350</y>
     <width>921</width>
     <height>141</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="write_all_btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>WRITE ALL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="write_selected_btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>WRITE SELECTED</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>41</width>
     <height>141</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="eye_btn">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>460</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_btn">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>420</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
